src/main/java/it/pagopa/pn/deliverypush/service/utils/WebhookUtils.java

```diff
@@ -203,4 +122,11 @@ public static boolean checkGroups(List<String> toCheckGroups, List<String> allow
     private static boolean isEmpty(List list){
         return list == null || list.isEmpty();
     }
+
+    public int getVersion (String version) {
+
+        String versionNumberString = version.toLowerCase().replace("v", "");
```

Comment:
gestire anche il caso di versione null/empty, ritornando il valore della v10.
nb: è una costante che vien letta da configurazione mi pare

Reply:
in questo caso torniamo il valore v10 come default? quale è il nome della costante?

Reply:
nel caso della categoriesByFilter quindi non è necessario prevedere il caso in cui la versione è v1 e i filterValues non sono vuote

Reply:
il nome della property è:
pn.delivery-push.webhook.first-version=v10

Nel caso in cui i filterValues sono specificati, vanno "onorati".
E' accettabile lasciare la logica del placeholder "DEFAULT" anche per la v1, tanto anche se non esisteva, nel caso in cui qualcuno spulci la documentazione della 2.3 e decida di usarla, vabbè riceverà quegli eventi li.

Reply:
come richiesto abbiamo effettuato le due modifiche cambiando il metodo categoriesByFilter e gestendo il caso null/empty nel metodo getVersion popolando la versione a v10 come default